packages/common/test/featureServiceHelpers.test.ts

```diff
@@ -6745,6 +6746,64 @@ describe("Module `featureServiceHelpers`: utility functions for feature-service
     });
   });
 
+  describe("_getDynamicFieldNames", () => {
+    it("will keep expression fields", () => {
+      const table = {
+        "name": "Cemeteries_1678394615385_StatsTable",
+        "sourceServiceName": "Cemeteries_9eb657cfc34545038ee388071b3f730f",
+        "sourceLayerId": 5,
+        "sourceLayerFields": [
+          {
+            "name": "burialcount",
+            "alias": "Burial Count",
+            "source": "parentglobalid",
+            "statisticType": "COUNT"
+          },
+          {
+            "name": "fullname1",
+            "alias": "Full Name 1",
+            "source": "fullname",
+            "statisticType": "MIN"
+          },
+          {
+            "name": "fullname2",
+            "alias": "Full Name 2",
+            "source": "fullname",
+            "statisticType": "MAX"
+          },
+          {
+            "name": "parentglobalid",
+            "alias": "Parent Global ID",
+            "source": "parentglobalid"
+          },
+          {
+            "name": "population",
+            "alias": "Population",
+            "source": "population"
+          },
+          {
+            "name": "area",
+            "alias": "Area",
+            "source": "area"
+          }, {
+            "name": "density",
+            "alias": "Density",
+            "expression": "population / area",
+            "type": "esriFieldTypeDouble",
+            "source": "population"
+          }
+        ],
+        "groupBy": "parentglobalid",
+        "materialized": false
+      };
+
+      const actual = _getDynamicFieldNames(table);
+
+      const expected = ["burialcount", "fullname1", "fullname2", "population", "area"];
```

Comment:
`area` looks like a source field, as opposed to `density`, which has an expression. What am I missing, please?